src/factories/Axis.ts

```diff
@@ -214,5 +219,17 @@ module n3Charts.Factory {
 
       return axis;
     }
+
+    cloneAxis(): D3.Svg.Axis {
```

Comment:
Hmmmm in this class, Axis when used in method names refers to the inner `axis` property. That's not really ideal, though...